### PR TITLE
App 89/ios logs

### PR DESCRIPTION
**Pull request by atar9500**

### CODE REVIEW

**Review by DanielZiv-SF — CHANGES_REQUESTED** [review]
please add rotation 15mb

**Comment by DanielZiv-SF** [comment]
and please add an example file

**Comment by atar9500** [comment]
Example log file
[notifications_logs.txt](https://github.com/welldone-software/react-native-notifications/files/5997095/notifications_logs.txt)

File: lib/ios/RNLogger.m [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#import <Foundation/Foundation.h>
+#import "RNLogger.h"
+
+@implementation RNLogger : NSObject
+
+NSString* BASE_DIR = @"%@/silverfort/logs/%@";
+NSString* LOGS_FILE_1 = @"notifications_logs.txt";
+NSString* LOGS_FILE_2 = @"notifications_logs_2.txt";
+unsigned long long FILE_LIMIT = 15 * 1024 * 1000;
```
Comment (DanielZiv-SF) [inline lib/ios/RNLogger.m]:
```suggestion
unsigned long long FILE_LIMIT = 6 * 1024 * 1000;
```
if after rotation we will send 2 files, then we should lower the max size (2 * 6 * 2)

**Review by DanielZiv-SF — APPROVED** [review]

**Pull request merged**